src/schema/entities.yaml

```diff
@@ -75,10 +75,9 @@ run:
   name: Run
   description: |
     If several scans of the same modality are acquired they MUST be indexed
-    with a key-value pair: `_run-1`, `_run-2`, `_run-3` etc. (only integers
-    are allowed as run labels).
+    with a key-value pair: `_run-1`, `_run-2`, `_run-3` etc. (only nonnegative
+    integers are allowed as run labels).
```

Comment:
```suggestion
    with a key-value pair: `_run-1`, `_run-2`, ..., `_run-<index>`
    (only nonnegative integers are allowed for the `<index>`).
```

Reply:
This puts the schema in conflict with the in-text definition. Should I add these changes to the in-text definition too?

Reply:
Didn't appreciate this would be a problem.  Happy to take this modification.